google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploy.java

```diff
@@ -198,37 +145,29 @@ public void exit(int exitCode) {
 
           if (deployOutput == null
               || deployOutput.getService() == null || deployOutput.getVersion() == null) {
-            consoleLogLn(
-                GctBundle.message("appengine.deployment.version.extract.failure") + "\n");
+            loggingHandler.print(
```

Comment:
I guess I'm thinking that perhaps the console output should also remain unlocalized, since the rest of the console output won't be.